e_os.h

```diff
@@ -301,10 +301,15 @@ struct servent *getservbyname(const char *name, const char *proto);
 # endif
 
 /* unistd.h defines _POSIX_VERSION */
-# if !defined(OPENSSL_NO_SECURE_MEMORY) && defined(OPENSSL_SYS_UNIX) \
+# ifndef OPENSSL_NO_SECURE_MEMORY
+#  if defined(OPENSSL_SYS_UNIX) \
      && ( (defined(_POSIX_VERSION) && _POSIX_VERSION >= 200112L)      \
```

Comment:
this might be indented one space more, otherwise ok.

Reply:
updated commit pushed.